assets/queries/cloudFormation/s3_bucket_access_to_any_principal/query.rego

```diff
@@ -1,29 +1,6 @@
 package Cx
 
-CxPolicy[result] {
-	resourceBucket := input.document[indexBucket].Resources[nameBucket]
-	resourceBucket.Type == "AWS::S3::Bucket"
-
-	policyStatements := [policyStatement |
-		resourcePolicy := input.document[indexBucket].Resources[_]
-		resourcePolicy.Type == "AWS::S3::BucketPolicy"
-		checkRef(resourcePolicy.Properties.Bucket, nameBucket)
-		policyStatement := resourcePolicy.Properties.PolicyDocument.Statement[_]
-	]
-
-	checkPolicyConfiguration(policyStatements)
-
-	publicAccessBlockConfiguration := resourceBucket.Properties.PublicAccessBlockConfiguration
-	publicAccessBlockConfiguration.RestrictPublicBuckets == false
-
-	result := {
-		"documentId": input.document[indexBucket].id,
-		"searchKey": sprintf("Resources.%s.Properties.PublicAccessBlockConfiguration.RestrictPublicBuckets", [nameBucket]),
-		"issueType": "IncorrectValue",
-		"keyExpectedValue": "'Resources.Properties.PublicAccessBlockConfiguration.RestrictPublicBuckets' is true",
-		"keyActualValue": "'Resources.Properties.PublicAccessBlockConfiguration.RestrictPublicBuckets' is false",
-	}
-}
```

Comment:
Why this rule was removed?

Reply:
Because the difference between both rules was the verification of `RestrictPublicBuckets` and `IgnorePublicAcls`. This "new rule" verifies both cases